src/js/media-controller.js

```diff
@@ -239,19 +240,20 @@ class MediaController extends MediaContainer {
     // TODO: Can we just reset all state here?
     // Should hasPlayed refer to the media element or the controller?
     // i.e. the poster might re-show if not handled by the poster el
-    this.propagateMediaState(MediaUIAttributes.MEDIA_PAUSED, true);
+    this.propagateMediaState(MediaUIProps.MEDIA_PAUSED, true);
   }
 
   propagateMediaState(stateName, state) {
-    const previousState = this.getAttribute(stateName);
+    const attrName = stateName.toLowerCase();
+    const previousState = this.getAttribute(attrName);
 
     propagateMediaState(this.mediaStateReceivers, stateName, state);
 
-    if (previousState === this.getAttribute(stateName)) return;
+    if (previousState === this.getAttribute(attrName)) return;
 
     // TODO: I don't think we want these events to bubble? Video element states don't. (heff)
```

Comment:
Want to go ahead and address this TODO for me? 😬 Feels pretty unnecessary to be bubbling all these events. (no pressure)

Reply:
The conclusion was no, this could be breaking, so too much for this PR.